src/main/kotlin/apply/ui/api/ApplicationFormController.kt

```diff
@@ -0,0 +1,44 @@
+package apply.ui.api
+
+import apply.application.ApplicationFormSaveRequest
+import apply.application.ApplicationFormService
+import apply.application.ApplicationFormUpdateRequest
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/register/application")
```

Comment:
```suggestion
@RequestMapping("/api/register/applications")
```

복수형태로 uri를 구성하는 편이 아래의 메서드에서 /{id}를 붙여 여러개의 자원 중 하나의 자원을 가져간다는 표시를 해줄 수 있을 거 같아요😃